model/directory_status.py

```diff
@@ -33,3 +48,18 @@ def message(self):
     @message.setter
     def message(self, message):
         self._message = message
+
+    def is_new(self):
+        return self.status == self.NEW
+
+    def is_invalid(self):
+        return self.status == self.INVALID
+
+    def is_partial(self):
+        return self.status == self.PARTIAL
+
+    def is_error(self):
+        return self.status == self.ERROR
+
+    def is_complete(self):
```

Comment:
Rather than these, why not just have an `equals` method or something?  So they can ask `directory_status.equals(DirectoryStatus.COMPLETE)`.  These `is_X` just seem a bit odd.